### PR TITLE
csranking: Chao Zhang, Qi Li, Hai-xin Duan

### PR DESCRIPTION
**Pull request by pkuzhangchao**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -2289,6 +2289,7 @@ Chao Tian,University of Tennessee,http://www.eecs.utk.edu/people/faculty/ctian1,
 Chao Wang 0001,University of Southern California,http://www-bcf.usc.edu/~wang626,q2L3v6kAAAAJ
 Chao Xu 0006,Peking University,http://www.cis.pku.edu.cn/faculty/vision/xuchao/xuchao01.htm,NOSCHOLARPAGE
 Chao Zhang 0001,Peking University,http://www.cis.pku.edu.cn/faculty/vision/zhangchao/zhangchao.htm,i-vJ_-UAAAAJ
+Chao Zhang 0008,Tsinghua University,http://netsec.ccert.edu.cn/people/chaoz,YreXlgsAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:2292]: This person does not appear to be in the CS department. Please read the CONTRIBUTIONS criteria and explain how this person (and others in this commit) are eligible for inclusion.

https://www.tsinghua.edu.cn/publish/csen/index.html

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (pkuzhangchao) [reply csrankings.csv:2292]: No, these three persons are not in CS department, but in the Institute for Network Sciences and Cyberspace, which is derived from the CS department two years ago, focusing on Network-related topics. This institue have its own graduate school, recruiting dozens of PhD and master students each year. The three persons added/updated here are all in tenure-track or tenured. 

The director of this institute is also the dean of the CS department, i.e., Prof. Jianping Wu. You could find detail information of this institute at the following address:
http://www.insc.tsinghua.edu.cn/publish/wyy/10784/index.html
There is no English version though (maybe Google Translate could help).

Comment (emeryberger) [reply csrankings.csv:2292]: This means they do not necessarily fit the inclusion criteria. Being able to supervise a PhD in a separate area when there is a CS department is not enough.

> Eligible faculty include only full-time, tenure-track
> faculty members on a given campus who can _solely_ advise PhD students in
> Computer Science. Faculty not in a CS department or similar who can
> advise PhD students in CS can be included regardless of their home
> department.

Comment (pkuzhangchao) [reply csrankings.csv:2292]: Does it mean the PhD students being advised must be in CS department? Regardless the fact that their major is computer science? There are several departments/institutes related to the CS major in Tsinghua University, as well as in many other universities.

Comment (emeryberger) [reply csrankings.csv:2292]: To be included, a faculty member *must* be able to *solely* advise a PhD student in the CS department.

Comment (pkuzhangchao) [reply csrankings.csv:2292]: hmm, not clear what is the point. It seems to me the only problem is the students are not officially in the CS department.

Comment (emeryberger) [reply csrankings.csv:2292]: The point is this: if they can't advise students in the CS department, they will not be eligible for inclusion in CSrankings.

Comment (pkuzhangchao) [reply csrankings.csv:2292]: OK, got it, it has to be literally Computer Science Department.

Then, the definition "CSRankings is a metrics-based ranking of top computer science institutions around the world" is literally an over-claim. Don't you think? Why not make it clear to all readers?

Something like "CSRankings is a metrics-based ranking of top Computer Science Departments **(literally, excluding other institutions)** around the world" is more appropriate.

**Comment by emeryberger** [comment]
Closing.

**Pull request closed without merging**